app/controllers/reminders/reminder/edit.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+
+  actions: {
+    saveReminder: function() {
+      this.transitionToRoute('reminders');
```

Comment:
So currently I believe if you console.log(this.get('model').get('hasDirtyAttributes')) the log would return true. Basically you're not saving your model after you edit it.